keras_retinanet/bin/evaluate.py

```diff
@@ -116,7 +117,7 @@ def main(args=None):
 
     # load the model
     print('Loading model, this may take a second...')
-    model = keras.models.load_model(args.model, custom_objects=custom_objects)
+    model = keras.models.load_model(args.model, custom_objects=custom_objects(args.backbone))
```

Comment:
I added a utility function for this: https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/models/__init__.py#L5

Reply:
@hgaiser  correct me if I'm wrong, but after the update you made, my PR is no longer needed. right?

Reply:
Err, it seems they did.